.Gruntfile.babel.js

```diff
@@ -175,6 +226,8 @@ module.exports = (grunt) => {
   //  Grunt Tasks
   // -------------------------------------------------------------------------
 
+  grunt.loadNpmTasks('grunt-svg-sprite')
```

Comment:
This should be removed (unnecessary, see `require('load-grunt-tasks')(grunt)`))

Reply:
You are right, removed.